src/infinity/libraries/TokenSwap/LibSwap.sol

```diff
@@ -167,6 +193,55 @@ library LibSwap {
         }
     }
 
+    /* ========== Smaller Specific Swap Functions ========== */
+
+    function swapToKlimaFromUsdc(uint256 sourceAmount, uint256 klimaAmount) internal returns (uint256 klimaReceived) {
+        address[] memory path = new address[](2);
+        path[0] = C.usdc();
+        path[1] = C.klima();
+
+        return _performToExactSwap(0, C.sushiRouter(), path, sourceAmount, klimaAmount);
+    }
+
+    function swapToKlimaFromOther(
+        address sourceToken,
+        uint256 sourceAmount,
+        uint256 klimaAmount
+    ) internal returns (uint256 klimaReceived) {
+        address[] memory path = new address[](3);
+        path[0] = sourceToken;
+        path[1] = C.usdc();
+        path[2] = C.klima();
+        return _performToExactSwap(0, C.sushiRouter(), path, sourceAmount, klimaAmount);
+    }
+
+    function swapWithRetirementBonds(
+        address sourceToken,
+        address carbonToken,
+        uint256 sourceAmount,
+        uint256 carbonAmount
+    ) internal returns (uint256 carbonRecieved) {
+        if (sourceToken == C.klima()) {
```

Comment:
I suggest also adding support for sKlima and wsKlima here out of the gate.

Reply:
If the source token is sKLIMA or wsKLIMA it gets changed to KLIMA on line 39.